### DISCUSSION TITLE
vrije-ruimte

### DISCUSSION THREAD

**Discussion by MokhtarAkle** [fdnd/docs.fdnd.nl#75] [Reflectie]
# Feedback op FDND agency

Tops: 
- Professionele werksfeer en houding
- Variatie aan projecten beschikbaar
- Veel zelfstandigheid
- Open communicatie tussen teams en docenten
- Flexibele planning
- Je wordt gemotiveerd om je best te doen door docenten en mede studenten
- Moeilijkheidsgraad zelf te kiezen

Tips:
- Meer verbinding leggen met het bedrijfsleven
- Meer opties voor vorm van projecten ( buiten al bestaande FDND projecten )
- Meer voorzieningen (schermen, werkplek, vergoeding)

**Comment by ju5tu5** [comment]
Opgenomen in de tekst
